.github/workflows/MainDistributionPipeline.yml

```diff
@@ -14,21 +14,21 @@ concurrency:
 jobs:
   duckdb-stable-build:
     name: Build extension binaries
-    uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@v1.3.2
+    uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@main
     with:
       extension_name: azure
-      duckdb_version: v1.3.2
-      ci_tools_version: v1.3.2
+      duckdb_version: main
```

Comment:
@samansmink do I leave this at `main` for now, refer to the current 1.4.0 branch, or sth else?